examples/chip-tool/commands/common/CHIPCommand.cpp

```diff
@@ -486,6 +490,8 @@ CHIP_ERROR CHIPCommand::InitializeCommissioner(CommissionerIdentity & identity,
             chip::Credentials::SetSingleIpkEpochKey(&sGroupDataProvider, fabricIndex, defaultIpk, compressed_fabric_id_span));
     }
 
+    CHIPCommand::sICDClientStorage.UpdateFabricList(commissioner->GetFabricIndex());
```

Comment:
I read the documentation being added in DefaultICDClientStorage nd I still have no idea what this function is for and why/when I should call it as an API consumer.

Reply:
updated, thanks